### ISSUE TITLE
Regarding the DataLoader and get_file function Issue

### ISSUE THREAD

**Issue by LinMu7177** [SunzeY/AlphaCLIP#57]
Thank you for your open source contribution.

I am trying to run the training code. Following the instructions in the "Data Preparation" section of the documentation, I prepared the GRIT-20M data and used train/data_prepare/sam_grit.py to generate files containing IDs and corresponding mask.

Then I noticed that the code in the get_file function in your code is missing. Could you please let me know if you forgot to include this part of the code, or if it is indeed expected for us to implement this functionality ourselves?

It seems like the data needs to be reorganized.

<img width="809" alt="image" src="https://github.com/user-attachments/assets/da9a6bc6-e8b3-486c-858c-9a1aad6632f8">
